dapr-http-server/src/Dapr/Server/HttpServer/Types.hs

```diff
@@ -0,0 +1,27 @@
+module Dapr.Server.HttpServer.Types where
+
+import Data.Aeson
+import Data.Map
+import Data.Text
+import GHC.Generics
+
+data SubscriptionInfo = SubscriptionInfo
+  { pubsubname :: Text,
+    topic :: Text,
+    route :: Text,
+    metadata :: Map Text Text
```

Comment:
`Map Text Text` Can we define a type for it like we defined in http-client? to reuse it later.

Reply:
Good suggestion. Let me refactor the code in next PR.